corehq/apps/cloudcare/middleware.py

```diff
@@ -1,7 +1,8 @@
+from django.conf import settings
 from django.utils.deprecation import MiddlewareMixin
 
-
 FORMPLAYER_SESSION_COOKIE_NAME = 'formplayer_session'
+FORMPLAYER_SESSION_COOKIE_HTTPONLY = settings.SESSION_COOKIE_HTTPONLY
```

Comment:
Not worth changing, but just curious what made you put this in a var rather than reference settings.SESSION_COOKIE_HTTPONLY directly?

Reply:
For me it felt like it represents a different concept, that whether the Formplayer session cookie should be `httponly` can be inferred based on whether the Django session cookie is `httponly`, but doesn't _have_ to be based on that. I also thought about just setting it `= True` or even using `True` directly in `set_cookie()` - I think in all instances it gets the same place.

Reply:
Just wanted to check that the reasoning was along those lines. I don't have great context here, but from an initial read of this setting it seems like isn't that different of a concept? This is a session cookie, and we are using the SESSION_COOKIE_HTTPONLY settings when setting the httponly flag? It doesn't help that we don't use this setting anywhere else, but it seems like we should probably be referencing this same setting in other places where we set session cookies right?

Reply:
I think it depends whether we're taking "session cookie" to mean "any cookie tied to the session" or "Django's `sessionid` cookie". I'm fine with the former interpretation, since in practice we do want it do be the same. https://github.com/dimagi/commcare-hq/pull/35150/commits/1301079fd23954e92d2deed1dd36f72331251431